en/docs/design/create-api/create-streaming-api/create-a-websocket-streaming-api.md

```diff
@@ -0,0 +1,159 @@
+                         </p>
+                         <div>
+                             <div class="confluence-information-macro-body">
+                                 <p>The API context is used by the Gateway to identify the API. 
+          Therefore, the API context must be unique. This context is the API's root context when invoking the API through 
+          the Gateway.</p>
+                             </div>
+                             <div class="confluence-information-macro confluence-information-macro-tip">
+                                 <span class="aui-icon aui-icon-small aui-iconfont-approve confluence-information-macro-icon"></span>
+                                 <div class="confluence-information-macro-body">
+                                     <p>You can define the API's version as a parameter of its context by 
+          adding the <code>{version}</code> into the context. For example, <code>{version}/websocket</code>. The API Manager 
+          assigns the actual version of the API to the <code>{version}</code> parameter internally. For example, 
+                                         <code>ws://localhost:9099/1.0.0/websocket</code>. Note that the version appears before the context, allowing you to 
+          group your APIs based on the versions.</p>
+                                 </div>
+                             </div>
+                         </div>
+                     </div>
+                 </td>
+             </tr>
+             <tr>
+                 <td colspan="2" class="confluenceTd">Version</td>
+                 <td colspan="1" class="confluenceTd">1.0.0</td>
+             </tr>
+             <tr>
+                 <td colspan="2" class="confluenceTd">Protocol</td>
+                 <td colspan="1" class="confluenceTd">
+                     <p>WebSocket</p>
+                 </td>
+             </tr>
+             <tr>
+                 <td colspan="2" class="confluenceTd">Endpoint</td>
+                 <td colspan="1" class="confluenceTd">
+                    <p>
+                    Use one of the following endpoints.
+                    <ul>
+                        <li>ws://echo.websocket.org:80</li>
+                        <li>wss://echo.websocket.org:443</li>
+                    </ul>
+                 </td>
+             </tr>
+         </tbody>
+     </table>
+             
+      [![Create WebSocket API Form]({{base_path}}/assets/img/learn/design-api/streaming-api/websocket/create-websocket-streaming-api-form.png)]({{base_path}}/assets/img/learn/design-api/streaming-api/websocket/create-websocket-streaming-api-form.png)
+
+4.  Click **CREATE** or **CREATE & PUBLISH** to create the API.
+
+     <html>
+     <div class="admonition note">
+     <p class="admonition-title">Note</p>
+     <p>The <b>CREATE & PUBLISH</b> option will only appear when a user who has <code>publisher</code> permission adds the details for the <b>Endpoint</b> field.</p>
```

Comment:
Similar to [1].
[1]https://github.com/wso2/docs-apim/pull/3284/files#r589208308